designs/virtualization/4.5/Scheduling/Scheduling-section/Scheduling-section.md

```diff
@@ -0,0 +1,18 @@
+---
+parent: Virtualization
+version: 4.5
+---
+
+# Scheduling Section
+
+Creating a new section in the VM details page, Overview tab
+('Scheduling': If, where and how this workload will be placed within the cluster's nodes)
+
+![Operator details page](img/vm-page-0-0.jpg)
+
+Introducing a new section within the Details tab: `Scheduling and resources requirements`.
```

Comment:
The capitalization of the section header isn't consistent with "Virtual Machine Details" -- which one is OpenShift proper?

Reply:
I see... I thought that there is a guideline to do sentence case but I see now that the current section titles are capitalized. I'll change it.